app/src/main/java/com/android/calendar/EventInfoFragment.java

```diff
@@ -497,6 +552,10 @@ public EventInfoFragment(Context context, Uri uri, long startMillis, long endMil
         // This may be used to explicitly show certain reminders already known
         // about, such as during configuration changes.
         mReminders = reminders;
+
+        if (mEventId == 0) {
```

Comment:
can you add a comment why this is needed?

Reply:
I'm not sure I understand. First there is `mIsTask` right before so could we use that instead of zero? Second there is a second constructor that sets the `mEventId` below. Which code path is used that it is not set?

Reply:
mEventId is 0 just for tasks cause in the table there is no relation to eventId, so we get the id from the URI

Reply:
why can't have an event the id 0? Why can't we use `mIsTask`?

Reply:
I fix this

Reply:
fixed